Mixpanel/MPNotification.m

```diff
@@ -28,7 +28,6 @@ - (instancetype)initWithJSONObject:(NSDictionary *)object {
         NSString *body = object[@"body"];
         if (![body isKindOfClass:[NSString class]]) {
             [MPNotification logNotificationError:@"body" withValue:body];
-            return nil;
```

Comment:
maybe don't log an error at all? since we expect it to be NSNull sometimes. or only error if it's not NSString and also not NSNull